Makefile

```diff
@@ -247,7 +247,7 @@ format:
 
 lint:
 	@echo "--> Running linter"
-	@go run github.com/golangci/golangci-lint/cmd/golangci-lint run
+	@go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.55.2 run
```

Comment:
Good call b/c this matches what is run in CI

https://github.com/celestiaorg/celestia-core/blob/39777b4a3aa7557b7d4b4d9401b29b4a553c8afa/.github/workflows/lint.yml#L38

Reply:
yeah, using their master branch will panic

although tbh idk why the ci linter started working again since it doesn't use this

Reply:
oh I thought you did this to reproduce the CI failure locally

Reply:
nah it was just useless without using the same version of the CI